Sources/Exporters/Zipkin/Implementation/ZipkinConversionExtension.swift

```diff
@@ -91,7 +91,7 @@ struct ZipkinConversionExtension {
                           kind: ZipkinConversionExtension.toSpanKind(otelSpan: otelSpan),
                           name: otelSpan.name,
                           timestamp: otelSpan.startEpochNanos / 1000,
-                          duration: ( otelSpan.endEpochNanos - otelSpan.startEpochNanos) * 1000000,
+                          duration: ( otelSpan.endEpochNanos - otelSpan.startEpochNanos) / 1000,
```

Comment:
while you are on this - maybe also check if it's greater than zero.

Reply:
duration is already unsigned, so no negative number here